dll/win32/oleaut32/oleaut.c

```diff
@@ -667,6 +668,9 @@ ULONG WINAPI OaBuildVersion(void)
 		FIXME("Version value not known yet. Please investigate it !\n");
 		return MAKELONG(0xffff, 40);  /* for now return the same value as for w2k */
     }
+#else
+    return MAKELONG(5014, 50); /* same value as for Win2K3 and later */
+#endif /* __REACTOS__ */
```

Comment:
I guess you can just add a `case 0x00000205:  /* Win2K3 */` to the switch above.

Reply:
It was the proposal in my patch. This would be easily be pushed upstream to Wine too (for Win2K3 compat)....

Reply:
Yes, I consider that:
 - If the return value other than Win2K3, it may be "incorrect" or "incompatible"
 - We're currently targeting Win2K3, so we needn't call `GetVersion`, and further, the same value also returned on Win11

If take them into account, add a `case 0x00000205:  /* Win2K3 */` and return `MAKELONG(0xffff, 50)` is recommended for this patch, I'd like to do it

Reply:
Imo, if it's possible to send a fix with this additional case to Wine without adding a nasty `#ifndef __REACTOS__`, that would be the better way. I believe that it would be accepted well, because you can prove it's correctness with a test program.

Reply:
Committed, and comment of this PR has updated